.agents/skills/aicr-triage/SKILL.md

```
@@ -153,6 +154,40 @@ An active board item absent from `$OPEN` is closed: report it under Manual Revie
+jq 'map({
+      number,
+      stallBase: ([ .createdAt,
+                    ( .comments[]
+                      | select((.body | test("^\\s*(Triaged|Closing):")) | not)
+                      | .createdAt ) ] | max)
+    })' "$OPEN" > "${TMPDIR:-/tmp}/aicr-triage-stall-${repo//\//-}.json"
+```
+
+The prefix is a reserved marker on this board, not free text: a substantive
+update never opens with `Triaged:`. Matching on author instead would leave the
+#870 case unfixed, since the comment there is a maintainer's.
```

markdownlint-cli2 (0.23.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)